openmetadata-service/src/main/java/org/openmetadata/service/airflow/AirflowRESTClient.java

```diff
@@ -37,6 +38,14 @@ public class AirflowRESTClient extends PipelineServiceClient {
   private static final String API_ENDPOINT = "api/v1/openmetadata";
   private static final String DAG_ID = "dag_id";
 
+  private static final Map<String, String> TYPE_TO_TASK =
+      Map.of(
+          PipelineType.METADATA.toString(), "ingestion_task",
+          PipelineType.PROFILER.toString(), "profiler_task",
+          PipelineType.LINEAGE.toString(), "lineage_task",
+          PipelineType.USAGE.toString(), "usage_task",
+          PipelineType.TEST_SUITE.toString(), "test_suite_task");
+
```

Comment:
We'll have to make sure we remember to add new workflow here (in addition to creating new workflow json schema